.github/CODEOWNERS

```diff
@@ -147,3 +147,26 @@ Dockerfile @juspay/hyperswitch-infra
 docker-compose.yml @juspay/hyperswitch-infra
 .dockerignore @juspay/hyperswitch-infra
 monitoring/ @juspay/hyperswitch-infra
+
+crates/router/src/core/revenue_recovery  @juspay/hyperswitch-revenue-recovery
+crates/router/src/core/payments/operations/proxy_payments_intent.rs @juspay/hyperswitch-revenue-recovery
+crates/router/src/core/payments/operations/payment_attempt_record.rs    @juspay/hyperswitch-revenue-recovery
+crates/router/src/core/revenue_recovery.rs  @juspay/hyperswitch-revenue-recovery
+crates/router/src/core/webhooks/recovery_incoming.rs  @juspay/hyperswitch-revenue-recovery
+crates/router/src/types/storage/revenue_recovery.rs  @juspay/hyperswitch-revenue-recovery
+crates/router/src/workflows/revenue_recovery.rs   @juspay/hyperswitch-revenue-recovery
+crates/hyperswitch_connectors/src/connectors/chargebee.rs  @juspay/hyperswitch-revenue-recovery
+crates/hyperswitch_connectors/src/connectors/stripebilling.rs  @juspay/hyperswitch-revenue-recovery
+crates/hyperswitch_connectors/src/connectors/chargebee/transformers.rs  @juspay/hyperswitch-revenue-recovery
+crates/hyperswitch_connectors/src/connectors/stripebilling/transformers.rs  @juspay/hyperswitch-revenue-recovery
+crates/hyperswitch_connectors/src/connectors/recurly.rs  @juspay/hyperswitch-revenue-recovery
+crates/hyperswitch_connectors/src/connectors/recurly/transformers.rs  @juspay/hyperswitch-revenue-recovery
```

Comment:
Would you end up adding more files to this list as and when you're integrating more connectors? Would have probably better to keep revenue recovery connectors in a separate module / folder in the connectors crate maybe.